apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryJob.cs

```diff
@@ -107,17 +107,19 @@ public BigQueryJob ThrowOnAnyError()
             var errors = Resource.Status?.Errors;
             if (errors?.Count > 0)
             {
-                throw new GoogleApiException(_client.Service.Name, $"Job {Reference?.ProjectId}/{Reference?.Location}/{Reference?.JobId} contained errors")
+                throw new GoogleApiException(_client.Service.Name)
                 {
                     Error = new RequestError
                     {
                         Errors = errors.Select(error => new SingleError
-                            {
-                                Location = error.Location,
-                                Reason = error.Reason,
-                                Message = error.Message
-                            })
-                            .ToList()
+                        {
+                            Location = error.Location,
+                            Reason = error.Reason,
+                            Message = error.Message
+                        }).ToList(),
+                        Message = $"Job {Reference?.ProjectId}/{Reference?.Location}/{Reference?.JobId} contained errors." +
+                        $"Message: {errors.First().Message}" +
+                        (errors.Count == 1 ? "" : $" And {errors.Count - 1} more errors.")
```

Comment:
I had the period in and removed it because I saw on the test that then we got two periods. But obviously it probably depends, some might come without.

I'll follow your proposal.